components/self-service/new-content-done.tsx

```diff
@@ -174,19 +224,20 @@ export default function NewContentDone() {
         Follow these steps:
       </Typography>
       <ul>
-        <li>If the above looks good, login to Jira <a href="https://issues.redhat.com/login.jsp?os_destination=%2Fdefault.jsp" target="_blank" rel="noopener noreferrer">here</a> (a separate tab will open) then click the "Create Jira" button below</li>
-        <li>In the "Create Issue" page, set the "Reporter" field as your UserId and click the "Create" button at the bottom</li>
-        <li>Share the ticket to <strong>@release-artists</strong> on <strong>#forum-ocp-art</strong> on Slack</li>
+        <li>If the above looks good, click SubmitRequest and a Jira and PR will be created</li>
+        <li>Inform <strong>@release-artists</strong> on <strong>#forum-ocp-art</strong> on Slack about the Jira and PR</li>
       </ul>
     </Box>
     <Box sx={{ py: 2 }}>
-      <Button
-        variant="contained"
-        onClick={handleCreateJira}
-        sx={{ mt: 1, mr: 1 }}
-      >
-        Create Jira
-      </Button>
+      {!isSubmitted && (
+        <Button
+          variant="contained"
+          onClick={handleSubmitRequest}
+          sx={{ mt: 1, mr: 1 }}
+        >
+          SubmitRequest
```

Comment:
```suggestion
          Submit Request
```
nit: add a space between the words

Reply:
fixed